### DISCUSSION TITLE
Link with parameter for pre-filled fields

### DISCUSSION THREAD

**Discussion by MrCasquette** [directus/directus#23056] [Feature Requests]
### Summary

Ability to create link or parameter like {{post_id}} to pre-fill an item creation

### Basic Example

I need to make it easy to reply to a comment. I create a field link button and just add the link to create a field with the pre-fill field parameter for example answer_to (the id of the comment I want to answer to)

### Motivation

Makes directus an easier platform for non developper user :) 

### Detailed Design

No design needed unless this feature became a full custom field, like the already existing button.

### Must Have:
- ability to target a field with a url params like ?answer_to=theidofmycomment

### Should Have:
- multiparameter compatibility

### Could Have:
- A full interface like the link button interface that already exist

### Won't Have:
- Less is more but feel free to adapt

### Drawbacks

I think there's no drawback with this feature, except perhaps that if users haven't developpers that know the concept they won't be able to fulfill these simple steps, unless the interface is made for user not developpers.

### Alternatives

I think today the only way to do this is extensions

### Adoption Strategy

I think this will not cause any issue with existing projects.

**Comment by MrCasquette** [comment]
Hey there is there any new on this feature ? Maybe an extension ?

**Comment by joggienl** [comment]
This discussion was mentioned in #23396, I am not sure if it relates completely but it seems to point to a specific solution that might solve the issue of creating items from a bookmark with a filter (like mentioned in #23396).

I would like to add to this discussion that it would work like this:

- I want to be able to pre-fill fields when creating an item via query parameters in the url
- I want to be able to create a item from a bookmark (preset) that respects the set filters (= adds those to the url so they get "pre-filled")

Additionally it would be nice, I think, to have that behaviour optionally via the bookmark (i.e. an option like `respect_filter`) because I can also see that you might not always want to set the filter automatically.
